guides/common/modules/proc_setting-the-service-level.adoc

```diff
@@ -1,9 +1,9 @@
 [id="Setting_the_Service_Level_{context}"]
 = Setting the service level
 
-You can configure an activation key to define a default service level for the new host created with the activation key.
-Setting a default service level selects only the matching subscriptions to be attached to the host.
-For example, if the default service level on an activation key is set to Premium, only subscriptions with premium service levels are attached to the host upon registration.
+You can configure an activation key to define a default service level for the new host created with it.
+Setting a default service level makes specific repositories available to the host.
+For example, if the default service level on an activation key is set to Premium, only repositories with premium service levels are available to the host after registration.
```

Comment:
I'm pretty sure repositories without a service level set will also be available. (It's only repos with non-matching service levels that would be hidden.)

```suggestion
For example, if the default service level on an activation key is set to Premium, repositories with premium service levels are available to the host after registration.
```